src/Calculator.Wasm/Program.cs

```diff
@@ -14,7 +15,10 @@ static void Main(string[] args)
 		{
 			Console.WriteLine("Program.Main");
 
-            Windows.UI.Xaml.Application.Start(_ => _app = new App());
+			FeatureConfiguration.UIElement.AssignDOMXamlName = true;
```

Comment:
What's for ? Is it a debug flag you included by error, or it's needed in release ? (I don't see anything in the PR which seems related to this)

Reply:
It's something very useful

Reply:
very useful ... for ? Is it only to propagate the `X:Name` in the DOM ?